### PR TITLE
Updated Security Headers

### PR DESCRIPTION
**Pull request by MartinBelton-gov**
Updated Security Headers

### CODE REVIEW

File: src/FamilyHubs.ReferralUi.Ui/Security/SecurityHeaders.cs [hunk 1]
```diff
@@ -106,12 +112,12 @@ public static IApplicationBuilder UseAppSecurityHeaders(this WebApplication app)
                     builder.AddBaseUri()
                         .Self();
 
-                    builder.AddFrameSrc()
-                        .From(new[]
-                        {
-                            "https://www.googletagmanager.com",
-                            "https://2673654.fls.doubleclick.net"
-                        });
+                    //builder.AddFrameSrc()
```
Comment (hardik-desai-kcl) [inline src/FamilyHubs.ReferralUi.Ui/Security/SecurityHeaders.cs:115]: delete commented out code

**Review by hardik-desai-kcl — CHANGES_REQUESTED** [review]

File: src/FamilyHubs.ReferralUi.Ui/Security/SecurityHeaders.cs (hunk 1 above)
Comment (MartinBelton-gov) [reply src/FamilyHubs.ReferralUi.Ui/Security/SecurityHeaders.cs]: Done

File: src/FamilyHubs.ReferralUi.Ui/Security/SecurityHeaders.cs [hunk 2]
```diff
@@ -68,32 +69,23 @@ public static IApplicationBuilder UseAppSecurityHeaders(this WebApplication app)
...
-                            "https://www.googletagmanager.com",
-                            "https://www.googleadservices.com",
-                            "https://googleads.g.doubleclick.net"
+                            "https://*.google-analytics.com/",
+                            "https://*.analytics.google.com",
+                            "https://*.googletagmanager.com"
                         })
                         // this is needed for GTM
-                        .UnsafeEval()
+                        //.UnsafeEval()
```
Comment (hardik-desai-kcl) [inline src/FamilyHubs.ReferralUi.Ui/Security/SecurityHeaders.cs:82]: delete this as well

File: src/FamilyHubs.ReferralUi.Ui/Security/SecurityHeaders.cs [hunk 3]
```diff
@@ -52,13 +53,13 @@ public static IApplicationBuilder UseAppSecurityHeaders(this WebApplication app)
...
-                            "https://www.google-analytics.com",
+                            "https://*.google-analytics.com",
                             /* application insights*/ "https://dc.services.visualstudio.com/v2/track", "rt.services.visualstudio.com/v2/track"
                         });
 
                     builder.AddFontSrc()
-                        .Self()
-                        .From(new[] { "https://fonts.gstatic.com" });
+                        .Self();
+                    //.From(new[] { "https://fonts.gstatic.com" });
```
Comment (hardik-desai-kcl) [inline src/FamilyHubs.ReferralUi.Ui/Security/SecurityHeaders.cs]: delete this as well

**Review by hardik-desai-kcl — CHANGES_REQUESTED** [review]

**Review by hardik-desai-kcl — APPROVED** [review]

**Pull request merged**